xgi/drawing/draw.py

```diff
@@ -739,7 +740,30 @@ def _color_arg_to_dict(arg, ids, cmap):
 
     Parameters
     ----------
-    arg : str, dict, iterable, or NodeStat/EdgeStat
+    arg : There are several different valid arguments. They are:
```

Comment:
Should we rename the variable `arg`? It does not feel very intuitive. `color`?
For brevity:
```suggestion
    arg : Several formats are accepted:
```

Reply:
I'll rename it to color_arg since it's not always a color